packages/node-app/src/utils/model.util.ts

```diff
@@ -44,6 +44,19 @@ export const setParticipantsStats = (giveaway: any) => {
   };
 };
 
+export const includeWinnersName = (giveaway: any) => {
+  const winners = giveaway.winners.map((winner) => {
+    const participantObj = giveaway.participants.find(
+      (p) => p.id === winner.id
+    );
+    return {
+      ...winner,
+      name: participantObj.name,
+    };
+  });
+  return { ...omit(giveaway, ['winners']), winners };
+};
+
```

Comment:
The one on top can also be optimize but since it is not a change from this PR I cannot comment there, so here it goes:
```javascript
export const giveawayStats = (giveaway: Giveaway) => {
  const initialStats = {
    nrConfirmedParticipants: 0,
    nrPendingParticipants: 0,
  };

  const stats = giveaway.participants.reduce((stats, participant) => {
    if (participant.state === ParticipantState.CONFIRMED) { 
      stats.nrConfirmedParticipants++;
    } else if (participant.state === ParticipantState.PENDING) { 
      stats.nrPendingParticipants++;
    }
    return stats;
  }, initialStats);

  return stats;
};
```